test/e2e/create_test.go

```diff
@@ -108,15 +108,15 @@ func (tc *testContext) waitForWindowsNodes(nodeCount int32, waitForAnnotations,
 	if expectError {
 		// The time we expect to wait, if the windowsLabel is
 		// not used while creating nodes.
-		creationTime = time.Minute * 5
+		creationTime = time.Minute * 10
 	} else {
 		creationTime = nodeCreationTime
 	}
 
 	// As per testing, each windows VM is taking roughly 12 minutes to be shown up in the cluster, so to be on safe
 	// side, let's make it as 20 minutes per node. The value comes from nodeCreationTime variable.  If we are testing a
 	// scale down from n nodes to 0, then we should not take the number of nodes into account. If we are testing node
-	// creation without applying Windows label, we should throw error within 5 mins.
+	// creation without applying Windows label, we should throw error within 10 mins.
```

Comment:
This comment should stay the same for the incorrect Windows label and you should add a line for the deletion case.